[*] Running ...
Traceback (most recent call last):
  File "/home/dokhanhnam1199/QD/problems/bpp_online/eval.py", line 112, in <module>
    avg_num_bins = -evaluate(dataset)
  File "/home/dokhanhnam1199/QD/problems/bpp_online/eval.py", line 55, in evaluate
    _, bins_packed = online_binpack(items.astype(float), bins)
  File "/home/dokhanhnam1199/QD/problems/bpp_online/eval.py", line 28, in online_binpack
    priorities = priority(item, bins[valid_bin_indices])
  File "/home/dokhanhnam1199/QD/problems/bpp_online/gpt.py", line 25, in priority_v2
    priority_score = np.where(np.abs(bins_remain_cap - item) > (bins_remain_cap / 2), 0.25 * priority_score, priority_score)
  File "/home/dokhanhnam1199/.local/lib/python3.10/site-packages/numpy/lib/_function_base_impl.py", line 1642, in interp
    return interp_func(x, xp, fp, left, right)
ValueError: setting an array element with a sequence. The requested array has an inhomogeneous shape after 1 dimensions. The detected shape was (4,) + inhomogeneous part.
8
1
140.1816079436383
78.34750920170268
110
